### DISCUSSION TITLE
Is there an easy way to create gt table with bar charts in the background?

### DISCUSSION THREAD

**Discussion by uriahf** [rstudio/gt#813] [Q&A]
I would like to do something like this with gt:

![image](https://user-images.githubusercontent.com/11351434/124605753-eab16900-de74-11eb-833a-d801dd346cd9.png)
https://rstudio.github.io/DT/010-style.html

**Comment by rich-iannone** [comment]
Something close to this can be (easily) done with `gtExtras`: https://jthomasmock.github.io/gtExtras/reference/gt_plt_bar.html. Otherwise, you'll have to prepare a function to generate HTML in each of the cells and use `fmt_passthrough()` or `fmt_markdown()` to ensure the HTML is not escaped.
